src/govuk/template.njk

```diff
@@ -43,7 +43,7 @@
     {% endblock %}
 
     {% block main %}
-      <div class="govuk-width-container">
+      <div class="govuk-width-container {{ containerClasses }}">
```

Comment:
Just a note that we'll need to add documentation to the Design System page template page.

Reply:
The issues says to document this in just the SassDocs for now: https://github.com/alphagov/govuk-frontend/issues/1567#issuecomment-532679373 Maybe documenting it in the Design System will be part of the SassDocs work.

Reply:
Yeah that's a good point. Created an issue: https://github.com/alphagov/govuk-design-system/issues/1075